### DISCUSSION TITLE
Combobox tags inside question

### DISCUSSION THREAD

**Discussion by kkamik** [microsoft/fluentui#32252] [Q&A] [answered]
Is there any way we can customize the ComboBox so that the tags are inside the input?

[Docs](https://fluent2.microsoft.design/components/web/react/combobox/usage):
![Screenshot 2024-08-07 at 8 11 40 AM](https://github.com/user-attachments/assets/0d62b962-efd0-4764-93f0-77a01bf17dab)

[Storybooks](https://react.fluentui.dev/?path=/docs/components-combobox--default#multiselect-with-tags):
![image](https://github.com/user-attachments/assets/4cdc118a-8961-4647-88eb-0918920f49e6)

**Comment by emmayjiang** [comment, marked as answer]
Hello! I think you may be looking for the [TagPicker](https://react.fluentui.dev/?path=/docs/components-tagpicker--default) component instead!

<img width="423" alt="Screenshot 2024-08-07 at 9 58 24 AM" src="https://github.com/user-attachments/assets/02ea649f-cdd4-4000-8589-86472cfdb5fc">
